build_r.R

```diff
@@ -115,6 +115,13 @@ for (src_file in c("lightgbm_R.cpp", "lightgbm_R.h", "R_object_helper.h")) {
   .handle_result(result)
 }
 
+result <- file.copy(
+  from = file.path("R-package", "inst", "make-r-def.R")
+  , to = file.path("lightgbm_r", "inst", "bin/")
```

Comment:
Not sure here but I remember

> This PR also proposes some simplifications to `build_r.R`. We were repeating the string literal `"lightgbm_r"` a lot, and I think it would be better to store it in one constant at the top of the script.
   https://github.com/microsoft/LightGBM/pull/3087#issue-418377725
```suggestion
  , to = file.path(TEMP_R_DIR, "inst", "bin/")
```